app/images/icons/README.md

```diff
@@ -0,0 +1,9 @@
+# Icon images
```

Comment:
This file would get included in the final production bundle. We could update the build script to exclude it, but it might be better to move this documentation somewhere else for now (the main readme? :thinking: ).

Anecdotally I have found that a lot of people do not notice readme files in sub-directories. The build system readme for example has escaped the notice of a few people trying to understand how that system works. It's not great to include it in the main readme either, because of how long it is, but :shrug:

Reply:
If you'd like to try updating the build script to exclude this so we can keep it here, here is where to look: https://github.com/MetaMask/metamask-extension/blob/develop/development/build/static.js#L113-L116

That entry copies this directory into the `dist` directory that becomes the production build. You can add a `pattern` to ensure only images are copied over. Be sure to test that it works with subdirectories too - I'm not sure how that pattern is used, or whether more complex patterns with `**` or negation would be supported.

Reply:
Hmm ok I'll just remove for now until it becomes an issue. I'm making a few assumptions here 😅

Reply:
Updated https://github.com/MetaMask/metamask-extension/pull/15983/commits/cb3594922aa909a575bf40ccd49b3fc42d167db0